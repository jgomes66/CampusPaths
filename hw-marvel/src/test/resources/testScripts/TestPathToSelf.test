# Simple test to see what happens when there is a path
# back to the starting character through others

LoadGraph g superheroWithPathToSelf.csv

FindPath g A B
FindPath g B C
FindPath g C A

ListNodes g

ListChildren g A
ListChildren g B
ListChildren g C

