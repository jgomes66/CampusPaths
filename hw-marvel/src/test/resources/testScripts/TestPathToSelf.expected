# Simple test to see what happens when there is a path
# back to the starting character through others

loaded graph g

path from A to B:
A to B via BookA
path from B to C:
B to C via BookA
path from C to A:
C to A via BookA

g contains: A B C

the children of A in g are: B(BookA) C(BookA)
the children of B in g are: A(BookA) C(BookA) C(BookB)
the children of C in g are: A(BookA) B(BookA) B(BookB)